<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_periode_form</class>
 <widget class="QDialog" name="import_periode_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>585</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>791</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Geolimi - Import des marées</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_url_fichier">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(209, 223, 245);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectionFichier">
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sélectionner le fichier à importer</string>
       </property>
       <property name="text">
        <string>Sélectionner un fichier</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Etape 1 : quantification des données à importer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLancementEtape1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Parcourir le contenu du fichier</string>
       </property>
       <property name="text">
        <string>Lancement étape 1</string>
       </property>
       <property name="icon">
        <iconset resource="resourcesGeolimi.qrc">
         <normaloff>:/plugins/geolimi/icons/import.png</normaloff>:/plugins/geolimi/icons/import.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pbEtape1">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Etape 2 : vérification des données à importer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLancementEtape2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Vérifier le contenu du fichier</string>
       </property>
       <property name="text">
        <string>Lancement étape 2</string>
       </property>
       <property name="icon">
        <iconset resource="resourcesGeolimi.qrc">
         <normaloff>:/plugins/geolimi/icons/import.png</normaloff>:/plugins/geolimi/icons/import.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pbEtape2">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_27">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Etape 3 : import</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLancementEtape3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Insérer les données dans la base</string>
       </property>
       <property name="text">
        <string>Lancement étape 3</string>
       </property>
       <property name="icon">
        <iconset resource="resourcesGeolimi.qrc">
         <normaloff>:/plugins/geolimi/icons/import.png</normaloff>:/plugins/geolimi/icons/import.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pbEtape3">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnQuitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
     <property name="icon">
      <iconset resource="resourcesGeolimi.qrc">
       <normaloff>:/plugins/geolimi/icons/quit.png</normaloff>:/plugins/geolimi/icons/quit.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resourcesGeolimi.qrc"/>
 </resources>
 <connections/>
</ui>
